examples/bri-3/src/bri/identity/bpiSubjects/api/subjects.controller.spec.ts

```diff
@@ -13,15 +13,21 @@ import { CreateBpiSubjectDto } from './dtos/request/createBpiSubject.dto';
 import { UpdateBpiSubjectDto } from './dtos/request/updateBpiSubject.dto';
 import { NAME_EMPTY_ERR_MESSAGE, NOT_FOUND_ERR_MESSAGE } from './err.messages';
 import { SubjectController } from './subjects.controller';
-import Mapper from '../../../utils/mapper';
+import { Mapper } from '@automapper/core';
+import { SubjectsProfile } from '../subjects.profile';
+import { AutomapperModule } from '@automapper/nestjs';
+import { classes } from '@automapper/classes';
 import { validate as uuidValidate, version as uuidVersion } from 'uuid';
 
 describe('SubjectController', () => {
   let sController: SubjectController;
+  let mapper: Mapper;
 
   beforeEach(async () => {
     const app: TestingModule = await Test.createTestingModule({
-      imports: [CqrsModule],
+      imports: [CqrsModule, AutomapperModule.forRoot({
+        strategyInitializer: classes(),
+      }),],
```

Comment:
yeah prettier throws errors

Reply:
Thanks @skosito  @Therecanbeonlyone1969 
Since this PR is going to be huge and having incremental commits as we discussed in slack, I'll be making sure that the final commit has all good formatting in place. 
Kindly review the changes from a module level change so that nothing is missed